changes/2521-layday.md

```diff
@@ -0,0 +1 @@
+Allow passing `json_encoders` in class kwargs (fixes #2521)
```

Comment:
```suggestion
Allow passing `json_encoders` in class kwargs
```

Issue get's automatically added to the line in history.

Reply:
done